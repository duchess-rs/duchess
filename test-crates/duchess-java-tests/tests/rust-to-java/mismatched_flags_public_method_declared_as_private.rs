duchess::java_package! {
    package flags;

    public class flags.Flags { //~ ERROR: member declared as `private`
        private int privateField;
        public flags.Flags();
        private int privateMethod();
        private int publicMethod();
    }
}

fn main() {}
