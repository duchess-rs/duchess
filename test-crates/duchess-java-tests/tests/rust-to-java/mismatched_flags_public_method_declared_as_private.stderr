error: error in class `flags.Flags`: member declared as `private` but it is `public` in Java, which appears in method `publicMethod()`
 --> tests/ui/mismatched_flags_public_method_declared_as_private.rs:4:5
  |
4 |     public class flags.Flags {
  |     ^^^^^^

error: aborting due to 1 previous error

